app.js

```diff
@@ -12,8 +15,14 @@ api.socket.initSocket({ socket: config.socket, wsType: config.ws_type, onNewMess
 setTimeout(init, 5000);
 
 function init() {
-  require('./server');
   try {
+    if (config.health_api && typeof config.health_api === 'number') {
```

Comment:
Add `health_api` to `fields` of `configReader` similar to

```
  debug_api: {
    type: Number,
  },
```

The condition will be `if (config.health_api) {`